packages/wallets/src/features/accounts/screens/IDVDocumentUpload/IDVDocumentUpload.scss

```diff
@@ -19,11 +19,11 @@
         @include mobile {
             padding: 0;
         }
-    }
 
-    & .wallets-dropdown,
-    & .wallets-textfield {
-        max-width: 68rem;
+        /* stylelint-disable-next-line selector-class-pattern */
+        & .deriv-input__container {
+            width: 100%; /* Remove after @deriv-com/ui dropdown component refactored */
+        }
```

Comment:
These are added as a temporary fix as the @deriv-com/ui dropdown component has an issue where only the dropdown list is full width and will be removed once the component is refactored

![image](https://github.com/binary-com/deriv-app/assets/142860499/598a1089-e1f9-422f-a31e-01749da54f5b)

Reply:
@heorhi-deriv this fix has been added into deriv-com/ui and package version has been updated